packages/communication-react/tsconfig.preprocess.json

```diff
@@ -10,7 +10,8 @@
       "@internal/calling-stateful-client": ["calling-stateful-client/src"],
       "@internal/chat-stateful-client": ["chat-stateful-client/src"],
       "@internal/react-components": ["react-components/src"],
-      "@internal/react-composites": ["react-composites/src"]
+      "@internal/react-composites": ["react-composites/src"],
+      "@internal/northstar-wrapper": ["northstar-wrapper/src"]
```

Comment:
This is for preprocess version, do we preprocess northstar as well?

Reply:
There's no conditional compiles for n*, and given we should not be introducing new usage of this library there shouldn't be in future so stable vs beta didn't need supported in this package

Reply:
(so I guess this line isn't needed? I just copy-pasted anywhere `@internal` packages were referenced)